baleen/export.py

```diff
@@ -169,60 +179,98 @@ def feedinfo(self, path):
         with open(path, 'w') as f:
             f.write(feeds.to_json(indent=2))
 
-    def export(self, root=None, categories=None, level=SAFE):
+    def export(self, root=None, categories=None, scheme=JSON, level=SAFE):
         """
-        Runs the export of the posts to disk.
+        Export all posts in categories to disk.
         """
-        self.root = root or self.root
+
+        root = root or self.root
+        categories = categories or self.categories
+
+        scheme = scheme or self.scheme
+        level = level or self.sanitize_level
+        if not self.valid_scheme(scheme):
+            raise ExportError(INVALID_EXPORT_SCHEME.format(scheme))
+
+        if not self.valid_sanitize_level(level):
+            raise ExportError(INVALID_SANITIZE_LEVEL.format(level))
+
+        self.initialize_export_directory(root)
+        category_filepaths = self.initialize_category_dirs(base_dir=root,    
+                                                           categories=categories)
 
         # Reset the counts object and mark export as started.
         self.counts = Counter()
         self.state = State.Started
 
-        # Make the directory to export if it doesn't exist.
-        if not os.path.exists(self.root):
-            os.mkdir(self.root)
+        # Iterate through all posts, writing them to disk correctly.
+        # Right now we will simply write them based on their object id.
+        for post, category in tqdm(self.posts(categories=categories),
```

Comment:
Hehe ... well I should have known that; the problem was that I was looking in the master branch not the develop branch and didn't see the import -- it's all good!